src/components/AssignmentSummary.jsx

```diff
@@ -44,91 +51,37 @@ class AssignmentSummary extends Component {
     }
   }
 
-  createMessage(text, texterId, assignmentId, contactCell) {
```

Comment:
I think bulk sending as a feature is going to need to be as separate as possible, so we can make sure this works for the US context.  This (and similar things with legal implications) probably need to be separated out to a new PR more than anything else, so we can see how it connects with the code.